Misc/NEWS.d/next/Build/2022-09-14-10-38-15.gh-issue-96821.Zk2a9c.rst

```diff
@@ -0,0 +1 @@
+Explicitly mark C modules that need defined signed integer overflow, and set up machinery for `-fstrict-overflow`.
```

Comment:
```suggestion
Explicitly mark C modules that need defined signed integer overflow,
and set up machinery for ``-fstrict-overflow``.
```

I assume you want to use a literal here, and not the default role (italic prose)

Reply:
My personal preference would be to name them _C **extension** modules_. Also, I think "machinery" is too vague. Why not just describe what this PR does?

Reply:
I'm not sure they are all extension modules?  Basically, I marked anything that's written in C that needs `-fno-struct-overflow`, no matter whether it's some extension or the core of the eval loop.

(The latter one has since been fixed, so that's no longer in this PR.  But it would have been marked.)

In any case, I implemented your suggestion.